config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_secretmanagersecrets.secretmanager.cnrm.cloud.google.com.yaml

```diff
@@ -308,32 +365,25 @@ spec:
                       type: string
                   type: object
                 type: array
-              createTime:
-                description: The time at which the Secret was created.
-                type: string
-              name:
```

Comment:
I think it's OK to drop create time, but `name` seems more likely to have some people reading it ... WDYT?

Reply:
My 2c: mark as deprecated, encourage people to read externalRef instead, say that we intend to remove `name` in the v1 API

Reply:
Marking it deprecation works for me. I assume people uses KCC mostly via GitOps, so the live status matters mostly when something goes wrong (and `name` doesn't weigh in too much unless it's server generated). But I like the idea of marking it deprecated, since this is a Beta resource